libraries/c_sdk/standard/ble/src/services/wifi_provisioning/iot_ble_wifi_provisioning.c

```diff
@@ -1432,9 +1433,14 @@ void _listNetworkTask( IotTaskPool_t taskPool,
         {
             if( strlen( scanNetworks[ idx ].cSSID ) > 0 )
             {
+                networks_found++;
                 _sendScanNetwork( IOT_BLE_WIFI_PROV_MSG_TYPE_LIST_NETWORK_RESP, &scanNetworks[ idx ] );
             }
         }
+        if(!networks_found)
```

Comment:
Minior code style error: need spaces within the brackets. if( !networks_found )

Reply:
fixed